### ISSUE TITLE
NTR: Uroporphyrin high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8859]
New term request Uroporphyrin high in urine (elthat_121124190005):   UMLS: 0235388

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000805
- label: Elevated urinary uroporphyrin level
- parents: [Elevated urinary porphyrin level (HP:6000535)](https://hpo.jax.org/app/browse/term/HP:6000535)
- pmid: 
- def: The amount of uroporphyrin in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8859
- synonyms:
